.github/workflows/release.yml

```diff
@@ -25,9 +25,14 @@ jobs:
         platform:
           - runner: ubuntu-22.04
             target: x86_64
+          - runner: ubuntu-22.04
+            target: aarch64
         python-version:
+          - "3.9"
```

Comment:
3.9 is pretty much EOL: https://endoflife.date/python. Do you need it @apcamargo ?

Reply:
I'd keep it. We still have 6 months until the EOL